app/views/assessments/invalid_link.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="auth-container">
+  <div class="row">
+    <div class="col-24 d-flex flex-column justify-content-center">
+      <div class="row">
+        <div class="col-md-15 col-lg-12 col-xl-10 mx-auto">
+          <div class="pb-4">
+            <p>It appears that this assessment link is invalid. Please make sure you're using the latest link sent.</p>
+            <p>If you believe this is a mistake, please contact your local health department.</p>
```

Comment:
Did we get this language approved?

Reply:
nope, I'll start something in the ui channel